zypp/PublicKey.h

```diff
@@ -25,6 +25,10 @@
 #include "zypp/Edition.h"
 #include "zypp/Date.h"
 
+typedef struct gpgme_data * gpgme_data_t;
+typedef struct _gpgme_key * gpgme_key_t;
+typedef struct _gpgme_subkey *gpgme_subkey_t;
```

Comment:
The typedef is not stricly required anyway, also using the const with the typedef'ed pointer is a bad idea because it introduces some const weirdness.  I think I just replace those with _gpgme_key * directly and forget about the typedef completely